library/src/shared/utility/currency-mask.ts

```diff
@@ -0,0 +1,8 @@
+export function CurrencyMask(value: number | null): number | null {
+  if (value && value.toString().includes('.')) {
+    const split = value.toString().split('.');
+    const maskedValue = split[0] + '.' + split[1].slice(0, 2);
```

Comment:
Why only show 2 after? Is this a `fiat` currency mask? If so, let's update the function name and a comment.

Reply:
Renamed to FiatMask and commented.